internal/github/github_test.go

```diff
@@ -138,3 +151,15 @@ func runCreatePrAndCaptureOutput() (bool, string, error) {
 
 	return didCreatePr, sb.String(), err
 }
+
+func runCreateDraftPrAndCaptureOutput() (bool, string, error) {
```

Comment:
Big fan of this over adding a 'draft' flag to the method above. Feels like a good place to favor DAMP over DRY 👍